### ISSUE TITLE
Mejorar el modelo de DQN

### ISSUE THREAD

**Issue by Facundo-Guarnier** [Facundo-Guarnier/Tesis-Implementacion#20]
Experimentar con capas Dropout y MaxPool en el modelo Keras. Por ejemplo:
- tf.keras.layers.Dropout(0.15)
- tf.keras.layers.MaxPool2D() 

Intentar hacer el entrenamiento con GPU y en memoria RAM y no CPU y disco.
